src/stimulus/data/encoding/encoders.py

```diff
@@ -1,4 +1,7 @@
-"""This file contains encoders classes for encoding various types of data."""
+"""This file contains encoders classes for encoding various types of data.  
+Encoders are classes that transfrom the raw data into a rapresentation that should be the exact input of the model.
+This mainly happen through enconding, padding, class convertion and other mathematical operations.
```

Comment:
```suggestion

```
I don't think this is needed, exact representation of the input is quite clear already